src/perfglue/CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-if(ALLOCATOR STREQUAL "tcmalloc" AND NOT WITH_SEASTAR)
+if(ALLOCATOR STREQUAL "tcmalloc")
```

Comment:
@liu-chunmei does the reason why we disabled heap profiler still hold ?

see 028159ef6ea0f43c4eaf2314d7d73f92bbc5bd99 .